projects/ExtraRepository.cmake

```diff
@@ -0,0 +1,33 @@
+foreach(extra_repository ${EXTRA_REPOSITORIES})
```

Comment:
Hmm, ok. But what about our closed content. Having this option makes it very easy to add this content into our internal builds.
Additionally we don't have to care about updating all the content in the separate directory. CMakes will do the job using this mechanism. I already tested this and it works like expected.
Honestly I wouldn't be happy about killing this code again :disappointed: